packages/react-ui-core/src/List/ListItem.js

```diff
@@ -0,0 +1,45 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+import themed from 'react-themed'
+
+@themed('*', { pure: true })
+
+export default class ListItem extends PureComponent {
+
+  static propTypes = {
+    theme: PropTypes.object,
+    className: PropTypes.string,
+    orientation: PropTypes.string,
+    listItemNodeType: PropTypes.string,
+    children: PropTypes.node,
+  }
+
+  static defaultProps = {
+    theme: {},
+    listItemNodeType: 'li',
+  }
+
+  render() {
+    const {
+      orientation,
+      theme,
+      className,
+      children,
+      ...props
+    } = this.props
+
+    return (
+      <this.props.listItemNodeType
```

Comment:
Let's destructure this above so it doesn't get passed into the spread props below.